sites/hashdev/src/components/CalculationBlock.tsx

```diff
@@ -1,9 +1,9 @@
 import { MockBlockDock } from "mock-block-dock";
+import { VoidFunctionComponent } from "react";
 
 const CalcBlock = require("calculation-block").default;
 
-// eslint-disable-next-line import/no-default-export
-export default () => (
+export const CalculationBlock: VoidFunctionComponent = () => (
```

Comment:
Non-blocking: What would you use `VoidFunctionComponent` for now? Seems to be the same as `FunctionComponent` (or better yet, `FC`)?

Reply:
This must be an artifact of me creating #771 first. I was going to merge that PR without upgrading to React 18 but then realised I could just have one PR instead and the diff would not be enormous. We have a mix of type aliases in the codebase anyway and we  can align this in a follow-up. It'd be also nice to introduce an ESLint rule that bans the usage of aliases. I’d prefer `FunctionComponent` as it’s more beginner-friendly but OK with `FC` too.

Reply:
I prefer `FunctionComponent` too. The only value of `FC` is brevity but "brevity over clarity" is not a principle we'd support anywhere else, I think.